### ISSUE TITLE
Anti Virus Scans

### ISSUE THREAD

**Issue by Dalethium** [xXBuilderBXx/MultiRPC#1]
As a hardcore skeptic of clickonce deployment applications, I took the liberty of manually downloading the application's .deploy files and scanning them:

https://www.virustotal.com/en/file/0fe08cb2dda49e4d8cdca355a5c950c2cada0f6c5d25d8d05410e506a32fcc5c/analysis/1548178243/

https://www.virustotal.com/en/file/4a2438ecfcad3e6e7bb942acf2c40fbe2c0d72e4982df303ab5828af26ca753e/analysis/1544546032/

https://www.virustotal.com/en/file/bbe8aa31bbdba1240571bbd13d477a20e846faae4d0596e20929c9b3b0110d92/analysis/1548052454/

https://www.virustotal.com/en/file/354c280d91ad0a8d56aca376f592c61862d4da5928729b50748d27f6ef04d01b/analysis/1547721245/

Perhaps you should release it as a zip for people as skeptical as me, if I didn't have time I would not have ended up using this application!

tl;dr; does not have a virus, should be released in other ways.

**Comment by xXBuilderBXx** [comment]
Yes i could just release a standalone exe package for it also the windows defender prompt is actually quite normal to show up. This is because the application does not have a code signing certificate and it costs like $100+ which is a joke
